spec/models/devise/token_authenticatable/model_spec.rb

```diff
@@ -51,6 +51,16 @@
         expect(entity.authentication_token).to eq(authenticated_entity.authentication_token)
       end
 
+      it "should authenticate with all the options passed in, not just the auth_token" do
+        conditions = {facebook_token: entity.facebook_token, auth_token: entity.authentication_token}
+        expected_conditions = {facebook_token: entity.facebook_token, authentication_token: entity.authentication_token}
+
+        expect(described_class).to receive(:find_for_authentication).with(expected_conditions).and_call_original
+
+        authenticated_entity = described_class.find_for_token_authentication(conditions)
+        expect(entity.authentication_token).to eq(authenticated_entity.authentication_token)
```

Comment:
Please, only use one expectation per example as this makes debugging easier. Write another example for the second expectation.